src/templates/_common/components/elements.jsx

```diff
@@ -240,9 +240,19 @@ export const DatePicker = ({ className, id, text }) => (
     </div>
 );
 
-export const Button = ({ children, className, href, id, text }) => (
-    <a id={id} href={href || undefined} className={className || undefined}>
-        { children || <span>{text}</span> }
+export const Button = ({ children, className, text_className, href, id, text, target, balloon }) => (
```

Comment:
can you break each variable into a new line?

Reply:
Noted and addressed.